code/src/App.js

```diff
@@ -1,9 +1,148 @@
+      </header>
+
+      {!showResult && (
+        <form onSubmit={handleSubmit}>
+          <div className='survey'>
+            <label for='name'>Fill in your first name, please</label>
+            <input
+              id='name'
+              type='text'
+              onChange={event => setName(event.target.value)}
+              required
+              value={name}
+            />
+
+            <div className='divider'>
+              * * * * * * * * * * * * * * * * * * * * *
+            </div>
+
+            <label for='nickname'>
+              Fill in your Christmas nickname, please
+            </label>
+            <input
+              id='nickname'
+              type='text'
+              onChange={event => setNickname(event.target.value)}
+              required
+              value={nickname}
+            />
+
+            <div className='divider'>
+              * * * * * * * * * * * * * * * * * * * * *
+            </div>
+
+            <h2>Which Christmas Character is your spirit animal?</h2>
+            {characters.map(group => (
+              <label className='radioButtons' key={group}>
+                <input
+                  type='radio'
+                  value={group}
+                  onChange={event => setCharacter(event.target.value)}
+                  checked={character === group}
+                />
+                {group}
+              </label>
+            ))}
+
+            <div className='divider'>
+              * * * * * * * * * * * * * * * * * * * * *
+            </div>
+
+            <h2>What do you prefer to eat?</h2>
+            {foods.map(group => (
+              <label className='radioButtons' key={group}>
+                <input
+                  type='radio'
+                  value={group}
+                  onChange={event => setFood(event.target.value)}
+                  checked={food === group}
+                />
+                {group}
+              </label>
+            ))}
+
+            <div className='divider'>
+              * * * * * * * * * * * * * * * * * * * * *
+            </div>
+
+            <label for='song'>Select your favourite Christmas Song</label>
+            <select
+              id='song'
+              onChange={event => setSong(event.target.value)}
+              required
+              value={song}
+            >
+              <option value=''>Select</option>
+              <option value='Jingle Bells'>Jingle Bells - Frank Sinatra</option>
+              <option value='Driving Home For Christmas'>
+                Driving Home For Christmas - Chris Rea
+              </option>
+              <option value='I Don´t Believe in Santa Claus'>
+                I Don´t Believe in Santa Claus - the Vandals
+              </option>
+              <option value='Last Christmas'>Last Christmas - Wham</option>
+              <option value='All I Want For Christmas Is You'>
+                All I Want For Christmas Is You - Mariah Carey
+              </option>
```

Comment:
Here you also can create an array with these values and use `map()` for rendering this